routers/register.router.ts

```diff
@@ -2,38 +2,38 @@ import { Router } from 'express';
 import { myDataSource } from '../config/database.configuration';
 import { User } from '../src/entities/User/User.entity';
 import bcrypt from 'bcryptjs';
-import {ValidationError} from "../utils/errorsHandler";
-import {staticText} from "../language/en.pl";
+import { ValidationError } from "../utils/errorsHandler";
+import { staticText } from "../language/en.pl";
 import { validatePassword } from '../utils/validatePassword';
 
 
 type UserRegiserData = {
   userId: string;
+  email: string,
   password: string;
   confirmPassword: string;
   registerToken: string | null;
 };
 
 export const registerRouter = Router()
   .post('/', async (req, res, next) => {
-    const { userId, password, confirmPassword, registerToken } = req.body as UserRegiserData;
+    // const { userId, password, confirmPassword, registerToken } = req.body as UserRegiserData;
+    const { email, password, confirmPassword } = req.body as UserRegiserData;
```

Comment:
Przy rejestracji wysyłamy userId, password, confirmPassowrd, registerToken. Ze wględów bezpieczeństwa musimy sprawdzać token podczas rejestracji bo tak to ktoś mógłby np. wysłac POST przez postmana i ominąć sprawdzanie tokena

Reply:
ok.